### ISSUE TITLE
question llm

### ISSUE THREAD

**Issue by kalle07** [praj2408/Langchain-PDF-App-GUI#2]
why not fully local ? like privateGPT or GPT4all?

**Comment by praj2408** [comment]
yeah we can run local gpt using llama3 and ollama. im currently working on this project and soon i'll upload it to github. please follow my github profile for more updates.
Thank you
